1001-2000/LIT1010Abuqer.xml

```diff
@@ -39,6 +52,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract>
+            <p>Gadl translated from Arabic by the efforts of abbā Salāmā (1348-1388); see 
+               colophon in <ref type="mss" corresp="EMML1826"/>,f. 49vb, <quote xml:lang="gez">ወጽህቀ፡ ባቲ፡ ለአዕልዎታ፡ እምዓረቢ፡ ለግዘ፡ አበነ፡ ቅዱስ፡ አባ፡ ሰላማ፡ ጸሎቱ፡ ትዕቀበነ፡ አሜን።=።</quote>; colophon in <ref type="mss" corresp="BLorient687"/>
```

Comment:
ለግዕዝ? አቡነ?

Reply:
I copypasted from Alessandro's file, so would leave it for now